testsuite/gnat2goto/lib/python/test_support.py

```diff
@@ -60,12 +57,23 @@ def process(debug, file, cbmcargs):
     if g2go_results.status != 0:
         print "ERROR code ", g2go_results.status, " returned by gnat2goto when translating " + unit
         print "CBMC not run"
-        return ""
+        return False
+
+    return True
+
+# only run the following if gnat2goto succeeded
+def process_cbmc(debug, files, cbmcargs):
+    """Process Ada file with gnat2goto and cbmc"""
+    jsout = []
+    for file in files:
+        unit          = os.path.splitext(file)[0]
+        file_jsout    = unit + ".json_symtab"
+        jsout.append(file_jsout)
 
-    # only run the following if gnat2goto succeeded
-    # Run(["cbmc", jsout, "--show-symbol-table"], output=symtab)
-    # Run(["cbmc", jsout, "--show-goto-functions"], output=gotoprog)
-    cmdline = ["cbmc", jsout]
+    out      = unit + ".out"
```

Comment:
This is pretty confusing now. You're just using the name of the last file as `out`?

Reply:
I guess I was. What about `TEST_NAME` instead of `unit` here?